test/web-element.test.js

```diff
@@ -270,9 +271,21 @@ describe("Web element tests", () => {
       var v = f.evaluate(fontFamily)
       expect(v).to.be.an.instanceOf(ElementAttributeValue)
       var h = v.getValue()
+      console.log(h);
```

Comment:
As a rule, unit tests should not write to the console, except temporarily if you are debugging something. So this line should be removed.

Reply:
i forgot to remove them, it was just for debug